### PR TITLE
[OSDOCS#11152]: Update to prereqs for configuring email notifications

### PR DESCRIPTION
**Pull request by laubai**
Version(s): 4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-11152

Link to docs preview:
https://78536--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_cluster_admin/osd-cluster-notifications.html
https://78536--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa-cluster-notifications.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/managed-cluster-add-notification-contacts.adoc [hunk 1]
```diff
@@ -12,8 +12,8 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Prerequisites
 * Your cluster is deployed and registered to the {hybrid-console}.
-* You are logged in to the {hybrid-console-second}.
-* The intended notification recipient has a user account on your cluster.
+* You are logged in to the {hybrid-console-second} as the cluster owner or as a user with the cluster editor role.
+* The intended notification recipient has a Red Hat Customer Portal account associated with the same organization as the cluster owner.
```
Comment (skopacz1) [inline modules/managed-cluster-add-notification-contacts.adoc:16]: Nit - per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non-breaking space between Red and Hat:
```suggestion
* The intended notification recipient has a Red{nbsp}Hat Customer Portal account associated with the same organization as the cluster owner.
```

File: modules/managed-cluster-add-notification-contacts.adoc [hunk 2]
```diff
@@ -27,3 +27,11 @@ By default, only the cluster owner receives cluster notification emails. You can
...
 .Verification steps
 * The "Notification contact added successfully" message displays.
+
+.Troubleshooting
+
+Error: `Account with ID <username> denied access to perform create on NotificationContact`::
+This error occurs when you are logged in to the cluster using an account that does not have the cluster owner or cluster editor roles. Log in to a different account that has these roles and try again.
+
+Error: `Could not find any account identified by <email-address>`::
+This error occurs when the intended notification recipient is not part of the same Red Hat account organization as the cluster owner. Contact your organization administrator to add the intended recipient to the relevant organization and try again.
```
Comment (skopacz1) [inline modules/managed-cluster-add-notification-contacts.adoc:37]: Same comment about nbsp:
```suggestion
This error occurs when the intended notification recipient is not part of the same Red{nbsp}Hat account organization as the cluster owner. Contact your organization administrator to add the intended recipient to the relevant organization and try again.
```

**Review by skopacz1 — COMMENTED** [review]
Just a few comments but otherwise LGTM!

**Comment by skopacz1** [comment]
Also friendly reminder to get this PR QE reviewed before requesting a merge review, or demonstrating QE approval if the review was done outside of the PR

**Comment by laubai** [comment]
@skopacz1 Thanks for the review and the reminder - I generally ask for QE review at the same time as peer review. Is that an issue? Should QE review happen before peer review as well?

Fixes applied

File: modules/managed-cluster-add-notification-contacts.adoc [hunk 3]
```diff
@@ -27,3 +27,11 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Verification steps
 * The "Notification contact added successfully" message displays.
+
+.Troubleshooting
+
+Error: `Account with ID <username> denied access to perform create on NotificationContact`::
+This error occurs when you are logged in to the cluster using an account that does not have the cluster owner or cluster editor roles. Log in to a different account that has these roles and try again.
+
+Error: `Could not find any account identified by <email-address>`::
```
Comment (jmekkatt) [inline modules/managed-cluster-add-notification-contacts.adoc:36]: @laubai Hi, The notification contact could be added with RedHat username  or email address. Hence the error message could be "**Could not find any account identified by <username |email-address>**". Kindly check.

File: modules/managed-cluster-add-notification-contacts.adoc [hunk 4]
```diff
@@ -27,3 +27,11 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Verification steps
 * The "Notification contact added successfully" message displays.
+
+.Troubleshooting
+
+Error: `Account with ID <username> denied access to perform create on NotificationContact`::
```
Comment (jmekkatt) [inline modules/managed-cluster-add-notification-contacts.adoc:33]: Hi @laubai  , The error message never indicated the user name but the account id belongs to user will show.
Btw, This error currently happens only when user granted with role "Cluster autoscaler editor" now and it is bug tracked under [OCMUI-729](https://issues.redhat.com/browse/OCMUI-729?focusedId=22947967&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22947967)  . For all other unsupported user roles , the notification option will be disabled with tool tip as below.
Screenshot from 2024-09-03 12-16-41

Hence not sure if we want to include the above definition as part of the doc definition. Please review it. Thanks

Comment (laubai) [reply modules/managed-cluster-add-notification-contacts.adoc:33]: Done, thanks for the heads up on that!

**Comment by jmekkatt** [comment]
Thanks @laubai for the changes. I will add lgtm.

**Pull request merged**